### DISCUSSION TITLE
Gist storage

### DISCUSSION THREAD

**Discussion by je-cook** [Fusion-Power-Plant-Framework/bluemira#600] [General]
This discussion is contains links to gists for code that doesn't need to live in the main repo

**Comment by je-cook** [comment]
A gist for a scipy-nlopt comparison test. The problem specified here is possibly multimodal. Passes with scipy 1.4.1 and nlopt 2.7.0 but fails with scipy 1.5.3. Unable to test with newer scipy version as of writing because of other dependencies

https://gist.github.com/je-cook/f513865a5fce5fcdf88d6d89321deb7b

**Comment by CoronelBuendia** [comment]
A gist for a bad direction cosine matrix:

https://gist.github.com/CoronelBuendia/75ada288579cb7f59eac67c2511bd55b

**Comment by CoronelBuendia** [comment]
A gist for the BLUEPRINT structural solver: https://gist.github.com/CoronelBuendia/6194c38bab5896939e400a6fb34890bb
and its tests: https://gist.github.com/CoronelBuendia/28ee051b944ad040e12cf26c6c879899

**Comment by CoronelBuendia** [comment]
A gist for the BLUEPRINT TF coil structure optimiser: https://gist.github.com/CoronelBuendia/2490b3c59d6034f3aa9e52c8c06500e7

**Comment by je-cook** [comment]
GS operator (or general) single function profiling: https://gist.github.com/je-cook/54d4a13b6f26afa03adf9dcf339a9fe0

**Comment by CoronelBuendia** [comment]
A gist for a FE material matrix that I can't remember how or why I made:
https://gist.github.com/CoronelBuendia/a7620bf04cad9a2501574010f43f1ba5

**Comment by je-cook** [comment]
A gist for finding the local minima of an array. Can find plateau's too. As discussed in #1487 not currently needed

https://gist.github.com/je-cook/7353bcf2e8f62684cdb31bf8cc7f8663

**Comment by CoronelBuendia** [comment]
A gist for comparison of approaches to calculating the maximum self-field in a solenoid
https://gist.github.com/CoronelBuendia/022a9b2e3fe81892b6b96c125d8c2037

**Comment by CoronelBuendia** [comment]
A gist for calculating FW areas in 2-D CSG model:

https://gist.github.com/CoronelBuendia/27d96fd7cab9d76ab07d492d181cb80c

**Comment by je-cook** [comment]
A gist of elliptical integrals written in numba'd python:
https://gist.github.com/timothy-nunn/d2b6f3cafe1ab23b05515acf39b6867b

**Comment by OceanNuclear** [comment]
A gist of PROCESS variables update script:
https://gist.github.com/OceanNuclear/05fa554eebd6b39d3f98ded18f1ac5bb
